ush/forecast_postdet.sh

```diff
@@ -645,13 +650,21 @@ WW3_postdet() {
     if [ ! -f ${waverstfile} ]; then
       echo "WARNING: NON-FATAL ERROR wave IC is missing, will start from rest"
     else
-      $NLN ${waverstfile} $DATA/restart.${wavGRD}
+      if [ $waveMULTIGRID = ".true." ]; then
+        $NLN ${waverstfile} $DATA/restart.${wavGRD}
+      else 
+        $NLN ${waverstfile} $DATA/restart.ww3
+      fi
     fi
   done  
 
-  for wavGRD in $waveGRD ; do
-    eval $NLN $datwave/${wavprfx}.log.${wavGRD}.${PDY}${cyc} log.${wavGRD}
-  done 
+  if [ $waveMULTIGRID = ".true." ]; then
+    for wavGRD in $waveGRD ; do
+      eval $NLN $datwave/${wavprfx}.log.${wavGRD}.${PDY}${cyc} log.${wavGRD}
```

Comment:
Can we do all these without using eval? I know that's how it was done before, but I'd like to reduce the number of evals in our scripts, not increase them.